doc/frameworks/xgboost/using_xgboost.rst

```diff
@@ -192,12 +192,14 @@ inference against your model.
 
 .. code::
 
+    serializer = StringSerializer()
+    serializer.CONTENT_TYPE = "text/libsvm"
```

Comment:
This is pretty ugly, but it'll be replaced with `LibSVM` serializer (#1776) when that gets merged.